ILRepack/Steps/ResourceProcessing/BamlStreamCollector.cs

```diff
@@ -54,6 +79,22 @@ public void Process(EmbeddedResource embeddedResource, ResourceWriter resourceWr
             }
         }
 
+        private void PatchGenericThemesBaml(ResourceWriter resourceWriter)
+        {
+            byte[] existingGenericBaml;
```

Comment:
Yes, I finished on friday the more complex use cases where we already have existing generic.xaml in different configurations (empty, w/ merged dictionaries and w/o merged dictionaries), but the state of the code was horrible, so will push a PR for that tomorrow ^_^